templates/includes/header.html

```diff
@@ -1,20 +1,19 @@
+        </a>
     </div>
+
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Menu</a>
+
+    <nav class="p-navigation__nav">
+      <ul id="main-navigation" class="p-navigation__links">
+        <li class="p-navigation__link"><a href="/install"{% if level_1 == 'install' %} class="active"{% endif %}>Install</a></li>
```

Comment:
Can you please add ```aria-current="page"``` to all active classes in the header, this will announce something like “Home, current page link”.